.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.6.3
```

Comment:
The latest Ruby version is 2.6.4, so can we update it here?

Reply:
Updated - thanks for the review!